### ISSUE TITLE
Fids tool should be updated to specify file names and automatically remove leakage

### ISSUE THREAD

**Issue by jadie1** [SCIInstitute/ShapeWorks#222]

**Comment by cchriste** [comment]
Related to #810, which will hopefully consolidate FIDS altogether.

**Comment by cchriste** [comment]
@jadie1 Can you please provide some more details? Even though #810 is clear, we don't understand what is being requested here. Thanks!

**Comment by jadie1** [comment]
@sheryjoe I am not remembering what this was about, can you provide more detail?

**Comment by sheryjoe** [comment]
That is a backlog now. We used to have a grooming workflow that converts meshes to distance transforms with sub-voxel accuracy using spatial partitioning (named fids). It is computationally expensive and results in sign inaccuracies with the mesh has sharp corners (e.g. clipped meshes), hence fixing leakage (what we used to name this behavior) was performed as a different step. As now we can optimize particles on meshes, we don't have to consolidate these tools.
